internal/auth0/connection/schema.go

```diff
@@ -323,6 +324,12 @@ var resourceSchema = map[string]*schema.Schema{
 					Optional:    true,
 					Description: "Icon URL.",
 				},
+				"ping_federate_base_url": {
+					Type:         schema.TypeString,
+					Optional:     true,
+					ValidateFunc: internalValidation.IsURLWithHTTPSorEmptyString,
```

Comment:
Given that this can be an empty string, would it make sense for one of the tests to have this value as an empty string?

Reply:
@Widcket Good callout, it seems the API doesn't allow an empty URL here, it only allows http or https URLs. Updated [`4d110bd` (#527)](https://github.com/auth0/terraform-provider-auth0/pull/527/commits/4d110bd9359a6fad1cbd5f533282d40e147a0996)